src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2136,12 +2135,13 @@ private static IEnumerable<Diagnostic> FilterDiagnosticsByLocation(IEnumerable<D
             }
         }
 
-        internal ImmutableArray<Diagnostic> GetDiagnosticsForSyntaxTree(
+        internal override ImmutableArray<Diagnostic> GetDiagnosticsForSyntaxTree(
             CompilationStage stage,
             SyntaxTree syntaxTree,
             TextSpan? filterSpanWithinTree,
             bool includeEarlierStages,
-            CancellationToken cancellationToken = default(CancellationToken))
+            bool includeDiagnosticsWithSourceSuppression,
```

Comment:
Why not make includeDiagnosticsWithSourceSuppression false by default (and includeEarlierStages true by default) like you did above?